laa_court_data_api_app/routers/laa_references.py

```diff
@@ -38,7 +38,7 @@ async def patch_maat_unlink(defendant_id: str, request: ExternalPatchRequest):
         return JSONResponse(status_code=400, content=LaaReferencesErrorResponse(
             error={'defendant_id': ['mismatch in ids given']}).dict())
 
-    logger.info(f"Calling_Maat_Patch_{defendant_id}")
+    logger.info("Calling_Maat_Patch", defendant_id=defendant_id)
```

Comment:
I know it's the convention you are using, and don't change it, BUT seeing MAAT as Maat is making my retinas itch

Reply:
😿